modules/transactions/src/main/java/org/apache/ignite/internal/tx/configuration/TransactionConfigurationSchema.java

```diff
@@ -28,23 +28,20 @@
  */
 @Config
 public class TransactionConfigurationSchema {
-    /** Default checking transaction interval. */
-    public static final long DEFAULT_ABANDONED_CHECK_TS = 5_000;
-
     /** How often abandoned transactions are searched for (milliseconds). */
     @Range(min = 0)
     @Value(hasDefault = true)
-    public final long abandonedCheckTs = DEFAULT_ABANDONED_CHECK_TS;
+    public final long abandonedCheckTs = 5_000;
 
-    /** Default transaction timeout (milliseconds). */
+    /** Default timeout for read-only transactions. */
     @Range(min = 1)
     @Value(hasDefault = true)
-    public final long timeout = 10_000;
```

Comment:
This still looks like a breaking change. Let's consult @ptupitsyn whether we are allowed to directly remove a configuration property

Reply:
Good catch. But why should we consult Pavel about changing the property we've added a couple of weeks ago?